Ginger/GingerCoreCommon/Repository/BusinessFlowLib/FlowControl.cs

```diff
@@ -256,7 +256,7 @@ public void CalculateCondition(BusinessFlow BusinessFlow, Environments.ProjEnvir
                 {
                     if (VE.Value.Contains("{Actual}"))
                     {
-                        if ((ARC.Actual != null) && StringManager.IsNumeric(ARC.Actual))
+                        if ((ARC.Actual != null) && ARC.Actual.All(char.IsDigit))
```

Comment:
in the old function we were checking char.IsDigit(c) && c != '.'
with the new condition check for != '.' is missing.

Flow control is very sensitive area, can we keep same conditions ?

Reply:
Done